compiler/rustc_interface/src/passes.rs

```diff
@@ -46,6 +45,7 @@ use std::any::Any;
 use std::cell::RefCell;
 use std::ffi::OsString;
 use std::io::{self, BufWriter, Write};
+use std::lazy::SyncLazy as Lazy;
```

Comment:
Let's avoid the rename here as well, and fix call-sites to use `SyncLazy`

Reply:
Fixed!